docs/reference/plugins.rst

```diff
@@ -182,6 +182,16 @@ Can the plugin be inserted inside the text plugin?  If this is ``True`` then
 
 See also: `icon_src`_, `icon_alt`_
 
+.. _system_plugin:
+
+system
+------
+
+Default: ``False``
+
+Plugins with this attribute set to True can be dropped in any placeholder, overriding
+any placeholder configuration in :setting:`CMS_PLACEHOLDER_CONF`.
```

Comment:
We can easily change the name of the attribute. I think this might be useful for special custom plugins and it does not really deliver "special powers".
I'm not good at naming, so I am open to less deceiving naming